ui/ui.gradle.kts

```diff
@@ -0,0 +1,195 @@
+import de.dynamicfiles.projects.gradle.plugins.javafx.JavaFXGradlePluginExtension
+import edu.wpi.first.wpilib.opencv.installer.Installer
+
+plugins {
+    `application`
+    id("javafx-gradle-plugin")
+    id("com.google.osdetector")
+}
+
+if (!(project.hasProperty("generation") || project.hasProperty("genonly"))) {
+    sourceSets {
+        test {
+            java {
+                exclude("**/ui/codegeneration")
+                exclude("**/ui/codegeneration/**")
+            }
+        }
+    }
+}
+
+dependencies {
+    compile(project(":core"))
+    compile(project(":ui:preloader"))
+    //ideProvider project(path= ":core", configuration= "compile")
+    compile(group = "org.controlsfx", name = "controlsfx", version = "8.40.11")
+    compile(group = "com.hierynomus", name = "sshj", version = "0.16.0")
+    compile(group = "org.apache.velocity", name = "velocity", version = "1.7")
+
+    val coreTestOutput = project(":core").dependencyProject.sourceSets["test"].output
+    testCompile(files(coreTestOutput))
+    testCompile(files(coreTestOutput.resourcesDir))
+    testCompile(group = "org.testfx", name = "testfx-core", version = "4.0.5-alpha")
+    testCompile(group = "org.testfx", name = "testfx-junit", version = "4.0.5-alpha")
+    testRuntime(group = "org.testfx", name = "openjfx-monocle", version = "1.8.0_20")
+    testCompile(group = "org.opencv", name = "opencv-java", version = "3.1.0")
+}
+
+evaluationDependsOn(":core")
+evaluationDependsOn(":ui:preloader")
```

Comment:
Oof, we need to fix those. Suuuper hacky. If it works for now, fine. But that's really frowned upon.

Reply:
These lines have been removed